### ISSUE TITLE
KeyError: '\n "wallpaper" on every preset or generated theme

### ISSUE THREAD

**Issue by kohotli** [dylanaraps/pywal#408]
```
λ wal -i ~/Pictures/joseph-barrientos-49318-unsplash.jpg
[I] image: Using image joseph-barrientos-49318-unsplash.jpg.
[I] theme: Set theme to _home_ko_Pictures_joseph-barrientos-49318-unsplash_jpg_dark_None_None_1.1.0.json.
[I] colors: Found cached colorscheme.
[I] wallpaper: Set the new wallpaper.
[I] sequences: Set terminal colors.




Traceback (most recent call last):
  File "/usr/bin/wal", line 11, in <module>
    load_entry_point('pywal==3.3.0', 'console_scripts', 'wal')()
  File "/usr/lib/python3.7/site-packages/pywal/__main__.py", line 208, in main
    parse_args(parser)
  File "/usr/lib/python3.7/site-packages/pywal/__main__.py", line 185, in parse_args
    export.every(colors_plain)
  File "/usr/lib/python3.7/site-packages/pywal/export.py", line 68, in every
    template(colors, file.path, join(output_dir, file.name))
  File "/usr/lib/python3.7/site-packages/pywal/export.py", line 15, in template
    template_data = "".join(template_data).format(**colors)
KeyError: '\n    "wallpaper"'
```

This is what I get when I try and generate a scheme. I get the same error even if I try generating on a different image. I'm running on ArchLinux that was last updated today.

**Comment by deviantfero** [comment]
what happens if you rename the image to just `unsplash.jpg` and try again?

**Comment by kohotli** [comment]
I get a different key error:

```
ko in ~/Pictures
λ wal -i unsplash.jpg
[I] image: Using image unsplash.jpg.
[I] colors: Generating a colorscheme.
[I] colors: Using wal backend.
[I] colors: Generation complete.
[I] wallpaper: Set the new wallpaper.
[I] sequences: Set terminal colors.




Traceback (most recent call last):
  File "/usr/bin/wal", line 11, in <module>
    load_entry_point('pywal==3.3.0', 'console_scripts', 'wal')()
  File "/usr/lib/python3.7/site-packages/pywal/__main__.py", line 208, in main
    parse_args(parser)
  File "/usr/lib/python3.7/site-packages/pywal/__main__.py", line 185, in parse_args
    export.every(colors_plain)
  File "/usr/lib/python3.7/site-packages/pywal/export.py", line 68, in every
    template(colors, file.path, join(output_dir, file.name))
  File "/usr/lib/python3.7/site-packages/pywal/export.py", line 15, in template
    template_data = "".join(template_data).format(**colors)
KeyError: '\n    "alpha"'
```

**Comment by kohotli** [comment]
now I'm getting
```
ko in ~
λ wal -i ~/Pictures/unsplash.jpg
[I] image: Using image unsplash.jpg.
[I] theme: Set theme to _home_ko_Pictures_unsplash_jpg_dark_None_None_6265917_1.1.0.json.
[I] colors: Found cached colorscheme.
[I] wallpaper: Set the new wallpaper.
[I] sequences: Set terminal colors.




Traceback (most recent call last):
  File "/usr/bin/wal", line 11, in <module>
    load_entry_point('pywal==3.3.0', 'console_scripts', 'wal')()
  File "/home/ko/.local/lib/python3.7/site-packages/pywal/__main__.py", line 208, in main
    parse_args(parser)
  File "/home/ko/.local/lib/python3.7/site-packages/pywal/__main__.py", line 185, in parse_args
    export.every(colors_plain)
  File "/home/ko/.local/lib/python3.7/site-packages/pywal/export.py", line 73, in every
    template(colors, file.path, join(output_dir, file.name))
  File "/home/ko/.local/lib/python3.7/site-packages/pywal/export.py", line 17, in template
    template_data = "".join(template_data).format(**colors)
KeyError: '\n\t\t"wallpaper"'
```
My other schemes are doing the same now.

**Comment by kohotli** [comment]
Okay I figured out the problem. I apparently had a bad file placed in ~/.config/wal/templates that was screwing things up. Removing everything from that directory fixed all my problems.

**Comment by indeedwatson** [comment]
> Okay I figured out the problem. I apparently had a bad file placed in ~/.config/wal/templates that was screwing things up. Removing everything from that directory fixed all my problems.

What is "bad file"? And how is deleting templates a solution?

I'm having the same error with dunstrc template, and I don't see how not using it is a solution, but I don't know how to really fix it.

**Comment by Shipwright0** [comment]
> I'm having the same error with dunstrc template, and I don't see how not using it is a solution, but I don't know how to really fix it.

Hey @indeedwatson ! I don't know if you are still having this problem, but I was having the same issue as you with my dunstrc file. 

I found that [this](https://github.com/dylanaraps/pywal/wiki/User-Template-Files#available-variables-and-syntax) section of the user templates was my problem. The variables that pywal uses are defined by curly brackets, and what I found is that there was a section of the default dunstrc that had curly brackets in the commented geometry section. This meant that those brackets needed to be escaped by using a second curly bracket.

Hope this helps you (or anyone else having the same problem in the future)!
